.github/workflows/rdev-delete-for-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: Delete rdev for PR
+
+on:
+  pull_request:
+    branches-ignore:
+      - staging
```

Comment:
`main` instead of `staging` now? I don't think we'll need the `staging` branch for the new CI/CD flow, since main will be deployed directly to staging env.  same in next workflow, below

Reply:
As written this workflow will only run if the base branch is not staging or prod. I think you mean we should remove `-staging` because that branch wont be used anymore and main will deploy directly to the staging environment?

Reply:
I guess it depends on what branch name is associated with the PR being closed. If it's the feature branch, then I think we want to ignore main. My main point is that we want to create and destroy rdev stacks only for feature branch PRs. So whatever that looks like here. :)

Reply:
So we only want rdevs for PRs when those PRs are targeting a feature branch(AKA not main), and not when the PR is targeting main or prod?

Reply:
No, he's saying that, in the case of a feature branch being merged into main, it depends on whether the branch name that gets fed into the GHA for such an action is `main` or `<feature_branch>`. If it is the former, then we want to ignore staging, because we don't want to try to find an rdev to delete when main is merging into staging; if it is the latter, then we want to ignore main, because main would be the branch name that would be passed to the GHA when main merges into staging.